zpm/publisher_s3.go

```diff
@@ -521,6 +585,14 @@ func (s *S3Publisher) publish(osarch *zps.OsArch, pkgFiles []string, zpkgs []*zp
 			Body:   metadataUp,
 		})
 
+		medatabaDbData := make([]byte, 0)
+		_, err = metadataDb.Read(medatabaDbData)
+		if err != nil {
+			return fmt.Errorf("unable to read downloaded file: %s", s.uri.Path)
+		}
+
+		eTag = md5.Sum(medatabaDbData)
```

Comment:
I almost sure it should be on lower levels, I see in newer version of sdk there is ETag field from UploadOutput https://github.com/aws/aws-sdk-go/blob/main/service/s3/s3manager/upload.go#L100-L115, maybe it makes sense to update sdk for zps, but it might be challenging :)

Reply:
Ok, I opened [an issue](https://github.com/fezz-io/zps/issues/28) so it can be addressed later.